addon/services/scroll-activity.js

```diff
@@ -88,6 +91,17 @@ export default Service.extend(Evented, FastBootCompatMixin, {
               subscriber.callback(scrollTop, subscriber.scrollTop);
             }
             subscriber.scrollTop = scrollTop;
+          } else if (scrollLeft !== subscriber.scrollLeft) {
+            // If the value is changing from an initial null state to a first
+            // time value, do not treat it like a change.
+            if (subscriber.scrollLeft !== null) {
+              if (!hasScrolled) {
+                run.begin();
+                hasScrolled = true;
+              }
+              subscriber.callback(scrollLeft, subscriber.scrollLeft);
+            }
+            subscriber.scrollLeft = scrollLeft;
```

Comment:
We need to set both scrollTop and scrollLeft every time, because it's possible that a user could scroll vertically and horizontally both in the same animation frame. If we don't set this, we could accidentally fire scroll events twice.